fluentlenium-core/src/main/java/org/fluentlenium/core/action/MouseActions.java

```diff
@@ -0,0 +1,23 @@
+package org.fluentlenium.core.action;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+
+public class MouseActions {
```

Comment:
I think it make sense to have separate class for Mouse and Keyboard Actions exactly like Selenium has. We are going to add D'n'D feature soon so this class will be extended shortly.